src/styles.scss

```diff
@@ -2,3 +2,37 @@
 
 html, body { height: 100%; }
 body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
+
+
+:root{
+    --primary-color:  rgba(132, 52, 225, 1);
+    --secondary-color:rgba(132, 52, 225, 1);
+    --white-color: white;
+    --accent-color: #000;
+    --gray-color: gray;
+  --padding: 10px;
+  --gap-small: 10px;
+  --gap-large: 20px;
+}
+
+
+.primary-color{
+    color: var(--primary-color);
+}
+
+.white-color{
+    color: var(--white-color);
+}
+
+
+@mixin flex-between{
+    display: flex;
+align-items: center;
+justify-content: space-between;
+}
+
+@mixin flex-center{
+    display: flex;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
I can see that you have a mixin here to handle the above suggestion, that is great but make this flexible and reusable , just create one mixin, let it receive parameters and assign it them e.g 
@mixin flex-option($display, $justify-content, $align-items,$direction) {
  display: $display;
  justify-content: $justify-content;
  align-items: $align-items;
  flex-direction: $direction;
}